src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -208,6 +212,9 @@ public ApiTestsFixture()
             this.stratisPowNode.ConfigParameters.Add("apiuri", "http://localhost:37221");
 
             this.InitializeTestWallet(this.stratisPowNode);
+            this.builder.StartAll();
+
+            Block.BlockSignature = true;
```

Comment:
` Block.BlockSignature = true;`
Are you sure this is correct?

Reply:
@dangershony The Block.BlockSignature has to do with the nodes that are started. It expects NetworkOptions.IsProofOfStake to be true for PoS otherwise the tests will fail. See https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/NBitcoin/NetworkOptions.cs I solve this by storing the initial value and restore it when the tests are done running as anything can be using that static. That also explains the dispose.

Reply:
Ideally we get rid of statics but that's my opinion about them.